src/stores/SegmentFeatureStore.ts

```diff
@@ -201,14 +205,17 @@ export class SegmentFeatureStore {
         const notificationStore = projectStore.notificationStore
         const basePath = this.projectStore.settingStore.basePath
         const segmentationData = segmentationStore.segmentationData
+        const markerStats: PlotStatistic[] = featuresToCalculate as PlotStatistic[]
```

Comment:
Won't have to do as PlotStatistic[] if featuresToCalculate is already a PlotStatistic[]